packages/tenacity-git/tenacity-git.pacscript

```diff
@@ -0,0 +1,109 @@
+name="tenacity-git"
+version="r23.g098a0225e"
```

Comment:
use `pkgver()` for auto-updates.

See [`awesome-git`](https://github.com/pacstall/pacstall-programs/blob/master/packages/awesome-git/awesome-git.pacscript) for an example.